build.gradle

```diff
@@ -12,7 +12,7 @@ apply plugin: 'signing'
 
 allprojects {
     group = 'com.cloudant'
-    version = '0.15.6-SNAPSHOT'
+    version = '1.0.0-SNAPSHOT'
```

Comment:
Does the version change belong here? At least a separate commit, as it's not directly related to the HTTP layer change.